support/debugtracer_amd64.go

```diff
@@ -0,0 +1,13 @@
+//go:build amd64 && debugtracer
```

Comment:
Since we're using `debugtracer_XXX.go `for these, I'd also rename the release tracer `.go` files to `tracer_XXX.go` (instead of `support_XXX.go`).

Reply:
> Since we're using `debugtracer_XXX.go `for these, I'd also rename the release tracer `.go` files to `tracer_XXX.go` (instead of `support_XXX.go`).

Renaming these Go files would cause conflicts with other commits, resulting in pipeline check failures. Therefore, I won't be renaming these Go files in this commit.